Resources/Prototypes/Loadouts/Jobs/Logistics/traitRestrictedGear.yml

```
@@ -0,0 +1,293 @@
+      traits:
+        - RestrictedGear
+    - !type:OverallTimeRequirement
+      min: 72000 # 20 hours
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutSalvageSpecialistWeapons
+    - !type:CharacterJobRequirement
+      inverted: true
+      jobs:
+        - SalvageSpecialist
+  items:
+    - Wakizashi
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)